k-distribution/k-tutorial/1_basic/02_basics/README.md

```diff
@@ -30,23 +28,43 @@ module LESSON-02-A
 endmodule
 ```
 
-Save this file and then run:
+Note that the name of the module and name of the file match. While the name of
+the module has to be written in capitals, the file name can have any 
+capitalization. It is not a requirement that the file carries the module
+name, but merely a convention. If you choose to name your file differenly, you will
+need to compile it with attribute `--main-module` followed by the name of
+the module, to point the compiler to the _main_ module of the definition. More 
+about naming conventions and main modules will follow in 
+[Lesson 1.5](../05_modules/README.md).
+
+Now, save the file and, from within the directory where you saved it, run:
 
 ```
 kompile lesson-02-a.k
 ```
 
-`kompile` is K's compiler. By default, it takes a program or specification
-written in K and compiles it into an interpreter for that input. Right now we
-are compiling a single file. A set of K files that are compiled together are
-called a **K definition**. We will cover multiple file K definitions later on.
-`kompile` will output a directory containing everything needed to execute
-programs and perform proofs using that definition. In this case, `kompile` will
-(by default) create the directory `lesson-02-a-kompiled` under the current
-directory.
+`kompile` is K's compiler. It takes a **K definition** and compiles it into an
+interpreter. Right now, you are compiling a single-file K definition, but 
+we'll cover multiple-file K definitions later on.
+
+You should get the following output:
+
+```
+[Warning] Compiler: Could not find main syntax module with name
+LESSON-02-A-SYNTAX in definition.  Use --syntax-module to specify one. Using
+LESSON-02-A as default.
+```
+
+It's just a warning, and you can ignore it for now. Compilation succeeded.
```

Comment:
Here, you can say that the compiler is reminding the user about the desirable convention or splitting definitions from implementation. That we'll teach them the proper way to do it in lesson 5, once the programs start to become bigger, but as these first examples are small, it ought to be fine to ignore them for now.